src/components/Router/HistoryRouter.tsx

```diff
@@ -87,6 +106,33 @@ export const HistoryRouter = (props: HistoryRouterProps) => {
     })
   }
 
+  const [passiveSignalsTracker, setPassiveSignalsTracker] = useState<
+    PassiveSignals.Tracker | undefined
+  >(undefined)
+
+  useEffect(() => {
```

Comment:
Maybe the Router is not the best place to start the tracker. Can't the `PassiveSignalsTracker` could wait for the sdkConfiguration using redux?

Reply:
I'm resolving this, we'll do it later when @DannyvanderJagt is done with its PR.